locales/base/translation.json

```diff
@@ -2823,5 +2823,26 @@
     "availableBalance": "Available: <0></0>",
     "selectToken": "Select token",
     "fiatPriceUnavailable": "Price unavailable"
+  },
+  "gasFeeWarning": {
+    "title": "You need more {{tokenSymbol}} for gas fees",
+    "descriptionMaxAmount": "Add {{tokenSymbol}} for gas fees or lower the amount you're {{action}}",
+    "descriptionNotEnoughGas": "Adjust the amount you're {{action}} or add {{tokenSymbol}} to continue",
+    "titleDapp": "You have an insufficient gas token balance",
+    "descriptionDapp": "Add {{tokenSymbol}} to complete this transaction",
+    "cta": "Buy {{tokenSymbol}}",
+    "ctaGasToken": "{{verb}} smaller amount",
+    "actions": {
+      "sending": "sending",
+      "swapping": "swapping",
+      "depositing": "depositing",
+      "withdrawing": "withdrawing"
+    },
+    "verb": {
```

Comment:
I suspect this is adding unnecessary complexity. With this change, the translations shown in the app would be a combination of two translation strings, which would require looking at both to determine tenses across our supported languages. 

The localization team typically focuses solely on the translation key, making this cross-check likely to result in errors. Instead, I suggest creating distinct description types, such as `descriptionNotEnoughGasSending`, `descriptionNotEnoughGasSwapping`, `descriptionNotEnoughGasDepositing`, and `descriptionNotEnoughGasWithdrawing`. The same would be done for `ctaGasToken` and `descriptionMaxAmount`.

This does result in more translation keys, but it's less error prone and easier for the localization team.